orchagent/aclorch.cpp

```diff
@@ -971,6 +994,26 @@ bool AclTable::create()
     attr.value.s32list.list = bpoint_list.data();
     table_attrs.push_back(attr);
 
+    if (type == ACL_TABLE_PFCWD)
+    {
+        attr.id = SAI_ACL_TABLE_ATTR_FIELD_TC;
```

Comment:
Just a thought.. we can have TC and STAGE outside the _if_ so, no need to duplicate the code below.